src/app/component-status/component-status.component.ts

```diff
@@ -170,10 +209,10 @@ export class ComponentStatusComponent implements OnInit {
   }
 
   private sortByComponentName(a: ComponentStatusItem, b: ComponentStatusItem) {
-    if (a.component < b.component) {
+    if (a.title < b.title) {
```

Comment:
Consider comparing using `a.title.localeCompare(b.title)` instead (a one liner)

Reply:
Good point, didn't know that one!